### PR TITLE
OpenPaaS-Suite/esn#25 Move css for loading animation

### LINKED ISSUE #25: Better loading screen for each SPA
#### Current behaviour

The current loading screen weirdly jumps around from the middle of the viewport to its top.

#### Suggested behaviour

Stop the jumping and add some nice loading effect (but no text should be allowed or https://github.com/OpenPaaS-Suite/esn-frontend-common-libs/issues/86 will happen).

### PR DESCRIPTION
**Pull request by pbson**
Partially fix: OpenPaaS-Suite/esn#25 and OpenPaaS-Suite/esn-frontend-common-libs#86

### CODE REVIEW

File: src/frontend/css/commons.less [hunk 1]
```diff
@@ -622,3 +622,49 @@ html:not(.ie9) .select::before, .select::before {
 .white-space-pre-line {
   white-space: pre-line;
 }
+
+/*
+ * Bouncing effect for loading animation
+ */
+.bounce {
```
Comment (tuanlc) [inline src/frontend/css/commons.less]: This deserves a dedicated file and the name should be more meaningful

Comment (itswillta) [reply src/frontend/css/commons.less]: `animations.less`?

Comment (tuanlc) [reply src/frontend/css/commons.less]: `animation` is the way (technical solution) that we use to create animation loading icons. The name should represent the feature (reasonable)

For example: `loading-spa-icon`

File: src/frontend/css/commons.less [hunk 2]
```diff
@@ -622,3 +622,5 @@ html:not(.ie9) .select::before, .select::before {
 .white-space-pre-line {
   white-space: pre-line;
 }
+
+
```
Comment (itswillta) [inline src/frontend/css/commons.less]: Redundant empty lines.

File: src/frontend/css/loading-icon.less [hunk 3]
```diff
@@ -0,0 +1,43 @@
+.bounce {
+  -webkit-animation: bounce 2s;
+  animation: bounce 2s;
+  -webkit-animation-iteration-count: infinite;
+  animation-iteration-count: infinite;
+}
+@-webkit-keyframes bounce {
```
Comment (itswillta) [inline src/frontend/css/loading-icon.less]: According to this https://developer.mozilla.org/en-US/docs/Web/CSS/WebKit_Extensions, we don't need to use `webkit` here anymore.

File: src/frontend/css/loading-icon.less [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+  0%,
+  25%,
+  50%,
+  75%,
+  100% {
+    -webkit-transform: translateY(0);
+    transform: translateY(0);
+  }
+  40% {
+    -webkit-transform: translateY(-20px);
```
Comment (itswillta) [inline src/frontend/css/loading-icon.less]: According to this https://developer.mozilla.org/en-US/docs/Web/CSS/WebKit_Extensions, we don't need to use `webkit` here anymore.

File: src/frontend/css/loading-icon.less [hunk 5]
```diff
@@ -0,0 +1,43 @@
+.bounce {
```
Comment (tuanlc) [inline src/frontend/css/loading-icon.less]: the class name should change: `loading-icon`

**Review by itswillta — APPROVED** [review]

**Review by tuanlc — APPROVED** [review]

**Review by piavgh — APPROVED** [review]

**Pull request merged**